<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictoHunterDialog</class>
 <widget class="QDialog" name="PictoHunterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Directory Comparison</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/qml/icons/ico-application.ico</normaloff>:/qml/icons/ico-application.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="hCheckBox">
     <property name="text">
      <string>*.h</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Src Code Dir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="srcCodeDirTextEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="selectSrcCodeDirButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/qml/icons/ico-bluefolder.png</normaloff>:/qml/icons/ico-bluefolder.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="cppCheckBox">
     <property name="text">
      <string>*.cpp</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="jsCheckBox">
     <property name="text">
      <string>*.js</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Exclusion List (picto inside listed directories will be considered as used pictos):</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QPushButton" name="addExclusionDirectory">
     <property name="text">
      <string>Add Exclusion Directory</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="qmlCheckBox">
     <property name="text">
      <string>*.qml</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Picto Dir:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pictoDirTextEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectPictoDirButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/qml/icons/ico-bluefolder.png</normaloff>:/qml/icons/ico-bluefolder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="17" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select Source Code Dir and Picto Dir</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QListView" name="exclusionListView"/>
   </item>
   <item row="16" column="0">
    <widget class="QPushButton" name="cleanPictoDirectoryButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Clean Picto Directory</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>srcCodeDirTextEdit</tabstop>
  <tabstop>pictoDirTextEdit</tabstop>
  <tabstop>selectSrcCodeDirButton</tabstop>
  <tabstop>selectPictoDirButton</tabstop>
  <tabstop>runButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
